gen_openj9_templates.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+
+# Remove old openj9 templates
+rm templates/openj9/*
+
+for original_template in templates/*.json; do
+  echo Creating OpenJ9 template for file "$original_template"
+  sed -e 's/OpenJDK/OpenJ9/' \
+    -e 's/sso74/sso74-openj9/' \
```

Comment:
```suggestion
    -e 's/sso\([0-9]*\)/sso\1-openj9/' \
```

Reply:
+1 on the suggestion from Hynek as it's more universal (would work regardless of current RH-SSO version) => no need to keep this updated in the future till it works